### ISSUE TITLE
Replace `ellipsoidal_to_cartesian` with ERFA function

### ISSUE THREAD

**Issue by astrojuanlu** [poliastro/poliastro#698]
Our `ellipsoidal_to_cartesian` is exactly equal to `gd2gce` from SOFA/ERFA in one simple test I did:

```
In [1]: a, b = 6378137.0, 6378137.0                                                                                                     

In [2]: lat, lon = 58, 25  # Arbitrary                                                                                                  

In [3]: f = (a - b) / a                                                                                                                 

In [4]: from poliastro.czml.utils import ellipsoidal_to_cartesian                                                                       

In [5]: from astropy import _erfa as erfa                                                                                               

In [6]: x, y, z = ellipsoidal_to_cartesian(a, b, lat, lon); x, y, z                                                                     
Out[6]: (753460.0733635349, -100606.81643041565, 6332677.773035965)

In [7]: x, y, z = erfa.gd2gce(a, f, lon, lat, 0); x, y, z                                                                               
Out[7]: (753460.0733635349, -100606.81643041565, 6332677.773035965)
```

**Comment by abhijeetmanhas** [comment]
@Juanlu001 I would like to work on the issue. Basically gd2gce converts geodetic coordinates to geocentric coordinates for a reference ellipsoid. 
I have to just replace function definition in `czml/utils.py` by 
`return erfa.gd2ce(a,(a-b)/a,lon,lat,0)` right? Should we keep the comments inside function same?

**Comment by Juanlu001** [comment]
Hi @abhijeetmanhas, thanks for your interest in poliastro! Yes, only changing that should be enough.

**Comment by astrojuanlu** [comment]
pyerfa was splitted! https://github.com/liberfa/pyerfa

**Comment by rafaellehmkuhl** [comment]
Juan, I can take this one as I think it was left.

We currently don't have PyERFA as a dependency, since it split from Astropy. Do you think it is ok to add it?

**Comment by astrojuanlu** [comment]
@vmos1 noticed that, for `a != b`, the results are not the same!

```
In [23]: lat, lon = 58, 25                                                                                                                                                                                                                    

In [24]: a, b = Earth.R.to(u.m).value, Earth.R_polar.to(u.m).value                                                                                                                                                                            

In [25]: x, y, z = ellipsoidal_to_cartesian(a, b, lat * u.deg, lon * u.deg); x, y, z                                                                                                                                                          
Out[25]: 
(<Quantity 3055830.60159627>,
 <Quantity 1424957.21154874>,
 <Quantity 5432271.27401853>)

In [26]: x, y, z = gd2gce(a, f, (lon * u.deg).to_value(u.rad), (lat * u.deg).to_value(u.rad), 0); (x * u.one, y * u.one, z * u.one)                                                                                                           
Out[26]: 
(<Quantity 3063227.38217292>,
 <Quantity 1428406.38697732>,
 <Quantity 5408966.60065562>)
```

**Comment by astrojuanlu** [comment]
This turned out to be not so trivial, since our code assumed geocentric coordinates. A fix is in progress in #995.

**Comment by astrojuanlu** [comment]
Fixed in #995.
